src/docs/asciidoc/en/firebirddocs/nbackup/firebird-nbackup.adoc

```diff
@@ -705,6 +729,21 @@ This parameter is only valid in combination with `-L` and when it is present, nb
 Because the size is given in pages, it has to be multiplied by the database page size in order to get the actual number of bytes to be copied.
 Or, if you use the `dd` copy utility, you could specify the page size as `(i)bs` and the output of `nbackup -L -S` as `count`.
 
+[[nbackup-lock-virtual-machine]]
+=== Locking/Unlocking database in case of Virtual Machines Backup
+
+Using Virtual Machines backup tools without preparing database for such type of backup can lead to the corrupted (i.e., useless) backup copy.
+
+Firebird server intensively uses its own cache in RAM to speed up operations, and implements complex techniques to ensure database consistency at the every given moment. Virtual Machine backup tools are not aware about Firebird's cache, and usually they do not consider database files as random-access files.
```

Comment:
Put sentences on their own lines

Reply:
This still has multiple sentences on a single line

Reply:
In paragraph about Virtual Machine backup? Please check again.